src/set_theory/game/pgame.lean

```diff
@@ -1020,6 +1020,110 @@ theorem lt_iff_sub_pos {x y : pgame} : x < y ↔ 0 < y - x :=
      ... ≤ y + 0 : add_le_add_left (add_left_neg_le_zero) _
      ... ≤ y : (add_zero_relabelling y).le⟩
 
+/-- The product of `x = {xL | xR}` and `y = {yL | yR}` is
+  `{xL * y + x * yL - xL * yL, xR * y + x * yR - xR * yR |`
+  `xL * y + x * yR - xL * yR, xL * y + x * yR - xL * yR}`. -/
+def mul : pgame → pgame → pgame
```

Comment:
Note the CI error: this function already exists in another file.

Reply:
Oh, I hadn't noticed, sorry for that.